tspp.java

```diff
@@ -171,4 +176,4 @@ public static void main(String[] args) {
 cost = 13
 
  */
- 
\ No newline at end of file
+ 
```

Comment:
Avoid making changes like this. They make the diff file cluttered.

Reply:
Sorry I will remember this from next time!